test/integration/authz_v1_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 	"istio.io/api/rbac/v1alpha1"
 	"istio.io/istio/pilot/pkg/model"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
doesn't need to specify v1 in the beginning as package name is v1

Reply:
Okay, removed it.